packages/frontend/src/Views/Home/Pagination/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { Fragment, useState } from "react";
+import {
+  Wrapper,
+  Title,
+  Tag,
+  RecipeTypeWrapper,
+  RecipeType,
+  Icon,
+  TagsWrapper,
+} from "./Accordion.styled";
+import { useTags } from "src/services/hooks/useTags";
+import plus from "src/assets/images/plus.svg";
+import minus from "src/assets/images/minus.svg";
+import { usePaginationContext } from "src/Views/Home/Pagination/hooks/usePaginationContext";
```

Comment:
wrzuć to do components  i możesz dodać alias do folderu component przykładowy masz już dodany w .tsconfig.paths.json